documentation/modules/ROOT/pages/_partials/knative-objects.adoc

```diff
@@ -98,19 +98,19 @@ kubectl --namespace {tutorial-namespace} get rev \
 --label=serving.knative.dev/service=greeter \
 --sort-by="{.metadata.creationTimestamp}"
 ----
-copyToClipboard::knative-revisions[]
+copyToClipboard::{doc-sec}knative-revisions[]
```

Comment:
i would like modify it for standard pattern i follow like `{doc-sec}-<clipboard id>`

Reply:
ok, gonna update my PR. the problem was that the copy to clipboard functionality was not working. pattern doesn't matter.

Reply:
updated now